tests/testing_utils.py

```
@@ -461,6 +462,8 @@ def wrapper(*args, **kwargs):
                     "pytest",
                     f"{file_path}::{func_name}",
                     "-sx",
+                    "--basetemp",
+                    f"/tmp/pytest-torchrun-{func_name}",
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/pytest-torchrun-"

(S108)